scrapers/default.py

```diff
@@ -670,15 +680,16 @@ def get_metadata(self, soup):
         rp = ScrapeHelper.div_class(soup.html.body, "frett-main", "reporter-profile")
         f = lambda tag: ScrapeHelper.general_filter(tag, "a", "class", "name")
         rname = rp.find(f) if rp else None
+        authname = None
         if rname:
-            rname = rname.string
+            authname = rname.string
         else:
             # Probably a blog post
             rp = ScrapeHelper.div_class(soup.html.body, "pistlar-author-profile-box")
             if rp and rp.h4:
-                rname = rp.h4.string
+                authname = rp.h4.string
 
-        author = rname if rname else "Ritstjórn mbl.is"
+        author: str = authname if authname else "Ritstjórn mbl.is"
```

Comment:
`author = authname or "Ritstjórn mbl.is"`

Reply:
Bad C/Dart habits creeping in.